eng/testing/performance/performance-setup.ps1

```diff
@@ -125,6 +125,10 @@ if ($RunFromPerformanceRepo) {
 }
```

Comment:
For the ps1, it appears to me that lines 124, 138, 143, 147, 155, 165&167, and 174 need such a check.  Is that the case for all of them?  Did I miss anything?